src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -612,30 +612,31 @@ int fx_muxer_t::parse_args_and_execute(
         }
 
         app_candidate = argv[cur_i];
-        bool is_app_runnable = (ends_with(app_candidate, _X(".dll"), false) || ends_with(app_candidate, _X(".exe"), false)) && pal::realpath(&app_candidate);
-        trace::verbose(_X("App %s runnable=[%d]"), app_candidate.c_str(), is_app_runnable);
-        // If exec mode is on, then check we have a dll at this point
-        if (exec_mode)
+        bool is_app_managed = (ends_with(app_candidate, _X(".dll"), false) || ends_with(app_candidate, _X(".exe"), false)) && pal::realpath(&app_candidate);
+        trace::verbose(_X("App candidate '%s' has %d managed executable extension"), app_candidate.c_str(), is_app_managed);
```

Comment:
I know this is only a verbose message, but I think it could be worded better if we switched on `is_app_managed`.

```
trace::verbose(
  is_app_managed ? 
         _X("App candidate '%s' has a managed executable extension") :
         _X("App candidate '%s' does not have a managed executable extension"),
  app_candidate.c_str());
```